ui/cryptomaterial/dropdown.go

```diff
@@ -184,7 +184,7 @@ func (d *DropDown) layoutOption(gtx layout.Context, itemIndex int) D {
 			}
 
 			dropdownItemWidth -= gtx.Dp(values.MarginPadding24) // account for the dropdown Icon
-			return item.Icon.Layout24dp(gtx)
+			return item.Icon.Layout20dp(gtx)
```

Comment:
I'm weary of changes to reusable components like this. It might make the component look the way you want on the part of the UI you're working on but would affect other parts of the UI using the component.

What's the motivation for this change?

Reply:
Icon size to text size. The size of the icon is way too big compared to the text. Makes the UI look like Johnny bravo.

Reply:
<img width="204" alt="image" src="https://github.com/crypto-power/cryptopower/assets/27733432/21b2c185-f93e-44bb-bb30-e4c903e04aef">
this is what it currently looks like and i still think the icon is still too big but its way better than using 24px.

Reply:
I see that none of the existing users of the Dropdown were using icon. It's probably why no one noticed that the size was too much.